.devcontainer/Dockerfile

```diff
@@ -0,0 +1,56 @@
+FROM docker.io/rust:1.56-bullseye
```

Comment:
i do think that the .github/workflows/ci.yaml MSRV check should be extended verify that the first 3 lines of this docker file contains the documented MSRV (to ensure this image is kept up to date)